app/models/concerns/foreman_templates/provisioning_template_import.rb

```diff
@@ -36,11 +40,15 @@ def import!(name, text, metadata)
         status  = template.update_attributes(data)
         result  = "  #{c_or_u} Template #{id_string}:#{name}"
         result += "\n    Operatingsystem Associations:\n    - #{oses.map(&:fullname).join("\n    - ")}" unless oses.empty?
-      elsif data[:operatingsystem_ids]
+        result += "\n    Organizations Associations:\n    - #{organizations.map(&:name).join("\n    - ")}" unless organizations.empty?
+        result += "\n    Location Associations:\n    - #{locations.map(&:name).join("\n    - ")}" unless locations.empty?
+      elsif data[:operatingsystem_ids] || data[:location_ids] || data[:organization_ids]
```

Comment:
probably better to use standard logger with different log levels, built-in ruby logger should be enough but I guess it's out of scope for this PR